### ISSUE TITLE
Trojan

### ISSUE THREAD

**Issue by goldentg** [PurityWasHere/Anomaly-Mod-Hosting#25]
openbullet.exe is infected with trojan

**Comment by 1tarko** [comment]
Yeah, exactly today vps slowness and only the openbullet running.

Purity disappeared

**Comment by 1tarko** [comment]
Yeah, exactly today vps slowness and only openbullet running.

Purity disappeared
